### PR TITLE
[Foreman 1.11] Fixes #12538 - Support rubocop 0.35.1

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

**Comment by stbenjam** [comment]
@daviddavis  What was the reason for closing this? Is it still needed?

**Comment by daviddavis** [comment]
@stbenjam I ran rubocop against master with 0.28 (or whatever the previous version was) and it looked like there were errors so I just assumed that you guys weren't checking your code with rubocop. Would be happy to reopen this though if you guys want to use rubocop in jenkins or locally. Let me know.

**Comment by daviddavis** [comment]
I take that back. It does look like your tests are running rubocop but they're using 0.28.0. Not sure how though...

**Comment by iNecas** [comment]
Given you changed the title, you probably figured out we run the tests against the foreman 1.10, as our release cycle is not synchonized with foreman (and we do releases that are usually compatible both with latest stable and nightly). What we could do is trying to get the changes that work both on old and new version now, and keeping the things that depend on the new rubocop version for later?

**Comment by daviddavis** [comment]
@inecas cool I updated the PR and commented out the 0.35.1 cops. When this project updates to foreman 1.11, we can just uncomment them and fix any breakages and then we should be good to go.

File: app/lib/actions/remote_execution/run_host_job.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ def plan(job_invocation, host, template_invocation, proxy, connection_options =
 
         raise _("Could not use any template used in the job invocation") if template_invocation.blank?
 
-        settings =  { :global_proxy   => 'remote_execution_global_proxy',
+        settings = { :global_proxy   => 'remote_execution_global_proxy',
                       :fallback_proxy => 'remote_execution_fallback_proxy' }
```
Comment (iNecas) [inline app/lib/actions/remote_execution/run_host_job.rb]: One space to the left would look slightly nicer :)

**Comment by iNecas** [comment]
Reopending new PR after rebase as #123

**Pull request closed without merging**